frontend/app/views/notes/_template.html.erb

```diff
@@ -10,18 +10,26 @@
 <% define_template "rights_restriction", jsonmodel_definition(:rights_restriction) do |form| %>
   <%= form.label_and_date("begin") %>
   <%= form.label_and_date("end") %>
-  <div class="control-group">
-    <%= form.label("local_access_restriction_type") %>
-    <div class="controls">
-      <% if form.readonly? %>
-        <ul>
-          <% Array.wrap(form.obj['local_access_restriction_type']).
-                   map{|type| I18n.t("enumerations.restriction_type.#{type}",type)}.
-                   sort.each do |type| %>
-            <li><%= type %></li>
-          <% end %>
-        </ul>
-      <% else %>
+  <% if form.readonly? %>
+    <%# ANW-1426: don't display local access restriction type if empty to prevent it from showing for conditions governing use notes, where it is inappropriate %>
+    <% unless form.obj['local_access_restriction_type'] == [] %>
```

Comment:
I think in cases where no values have ever been set for `local_access_restriction_type` the property will be `nil` rather than `[]`. So maybe better to check that it is present and not empty.